cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1137,3 +1138,25 @@ def test_mixed_security_group_overwrite_validator(head_node_security_groups, que
     )
     expected_message = "make sure.*cluster nodes are reachable" if expect_warning else None
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "root_volume_size, expected_message, ami_size",
+    [
+        (65, None, 50),
+        (25, None, 8),
```

Comment:
Why should we test `root_volume_size=25,ami_size=8` if we already have `root_volume_size=65,ami_size=50`?

Reply:
I removed one of them